src/lib.rs

```diff
@@ -208,6 +222,18 @@ impl<'a> Ammonia<'a> {
             child.parent.replace(None);
             dom.append(&parent.clone(), NodeOrText::AppendNode(child.clone()));
         }
+        pass
+    }
+
+    fn fix_child(&self, child: &mut Handle, link_rel: &Option<StrTendril>) {
```

Comment:
Oh right (my brain is already gone on vacation :D). Then why are you not converting those in `fix_child` ?